app/vitest.config.ts

```diff
@@ -6,7 +6,7 @@ import viteConfig from './vite.config'
 import dotenv from 'dotenv'
 
 dotenv.config()
-
+process.env.TZ = 'UTC'
```

Comment:
is this okay  @hermannleboss ? i need to do this because there is time different in CI and in the local testing